src/ocean/util/app/ext/ReopenableFilesExt.d

```diff
@@ -96,7 +96,7 @@ public class ReopenableFilesExt : IApplicationExtension, ISignalExtExtension
 
     public this ( SignalExt signal_ext = null, int reopen_signal = SIGHUP )
     {
-        if (signal_ext)
+        if (signal_ext && reopen_signal)
```

Comment:
Would it make sense to also update the doc for `reopen_signal` to describe this usage?

Reply:
Yes, I suppose so. But I don't know what's the best way of updating the doc. Let me push a fixup commit as a starting point.